symbology/custom-dictionary-style/README.md

```diff
@@ -10,27 +10,33 @@ When symbolizing geoelements in your map, you may need to convey several pieces
 
 ## How to use the sample
 
-Pan and zoom the map to see the symbology from the custom dictionary style.
+Use the radio buttons to toggle between the dictionary symbols from the web style and style file. Pan and zoom around the map to see the symbology from the chosen dictionary symbol style.
```

Comment:
```suggestion
Use the radio buttons to toggle between the dictionary symbols from the web style and style file. Pan and zoom around the map to see the symbology from the chosen dictionary symbol style. The web style and style file are slightly different to each other to give a visual indication of the switch between the two.
```

Reply:
Bit of unsolicited review there :D. Not sure if the sentence I've add reads well, but just something to clue the user in to that expected difference when using one style vs the other